src/Files.App/Views/Settings/AboutPage.xaml.cs

```diff
@@ -17,5 +21,17 @@ public AboutPage()
 
 			ViewModel = new AboutViewModel();
 		}
+
+		private void ThirdPartyLicenses_Tapped(object sender, TappedRoutedEventArgs e)
```

Comment:
Is this called when the control is loaded?

Reply:
This is called when Third party licenses SettingsBlockControl is clicked and MarkdownTextBlock is to expand.

Reply:
There isn't Expanded event?

Reply:
Seems not. It did not appear in IntelliSense.

Reply:
You can use Click event, instead.

https://github.com/files-community/Files/blob/3712ddf262fd37147b306682e270dc9c6fefc211/src/Files.App/UserControls/Settings/SettingsBlockControl.xaml.cs#L133-L141

Reply:
@onein528 Thank you! Fixed.